cash/cash.c

```diff
@@ -0,0 +1,44 @@
+#include <cs50.h>
+#include <stdio.h>
+
+int main(void)
+{
+    int change;
+
+    do
+    {
+        change = get_int("Change owed: ");
+    }
+    while (change <= 0);
+
+    int quarters = 0;
+    int dimes = 0;
+    int nickels = 0;
+    int pennies = 0;
+
+        while (change >= 25)
```

Comment:
this violates styling requirements from CS50. Please do your changes in VSCode online and then use `style50` and `check50` utilities to ensure your code is 100% valid